aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -38,6 +39,34 @@ use std::{
     sync::Arc,
 };
 
+/// Sporadically checks if the given two input type layouts match.
+pub fn randomly_check_layout_matches(
```

Comment:
@vusirikala  - can you also address this TODO to use this function:

https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/block-executor/src/executor.rs#L859

you might need to move this function to a more generic file, that can be called throughout the code.

Reply:
I tried to put the `randomly_check_layout_matches` function in value.rs where MoveTypeLayout was defined. But that crate is using an older version of rand and I got some errors due to incompatibility issues. Is it okay if I continue keeping the `randomly_check_layout_matches` in change_set.rs and use the function in executor.rs?

Reply:
if you can refer it there from here, that's fine for now